tools/umi_tools/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM continuumio/miniconda3:4.8.2
```

```suggestion
FROM nfcore/base:1.7
```

See comment below about procps. Also allows us to easily fix future stuff across all modules.